flink-connector-debezium/src/main/java/com/ververica/cdc/debezium/JsonDebeziumDeserializationSchema.java

```diff
@@ -46,35 +47,53 @@ public class JsonDebeziumDeserializationSchema implements DebeziumDeserializatio
      * schema in messages.
      */
     private final Boolean includeSchema;
+    private final int timeZoneOffset;
 
-    /** The custom configurations for {@link JsonConverter}. */
+    /**
+     * The custom configurations for {@link JsonConverter}.
+     */
     private Map<String, Object> customConverterConfigs;
 
     public JsonDebeziumDeserializationSchema() {
         this(false);
     }
 
     public JsonDebeziumDeserializationSchema(Boolean includeSchema) {
+        this(includeSchema, 8);
```

Comment:
Thanks @youyangkou for the contribution, but the PR looks like work in a wrong way. 
Firstly, the  timeZoneOffset `8` works is  that you happen to be in GMT+8 timezone, for other timezone it doesn't work.
Secondly, Not all source need consider the timezone issue, for example, pg doesn't exist the timezone issue, and even in MySQL, only the record from changelog need consider the timezone offset.